src/add-score.js

```diff
@@ -0,0 +1,56 @@
+function addScore(name, score, id) {
+  const ScoresContainer = document.getElementById('leaders-container');
+  const bookHTML = document.createElement('div');
+  bookHTML.classList.add('leader-item');
+  if (id % 2 === 0) {
+    bookHTML.classList.add('gray');
+  }
+  bookHTML.innerHTML = `
+                <p class="name">${name}:</p>
+                <p class="score">${score}</p>
+            `;
+  ScoresContainer.appendChild(bookHTML);
+}
+
```

Comment:
- [ ] Kindly use the arrow function instead of the regular function. [Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Use%20arrow%20functions%20instead%20of%20the%20function%20keyword.)